packages/core/content-manager/server/src/controllers/relations.ts

```diff
@@ -89,15 +86,17 @@ export default {
     }
 
     const where: Record<string, any> = {};
-    if (!isSourceComponent) {
+    if (!isSourceComponent && locale) {
       where.locale = locale;
-      where.publishedAt = status === 'published' ? { $ne: null } : null;
```

Comment:
should the where.publishedAt be here instead of below regarldess of an id being present or not?